lib/season_stats.rb

```diff
@@ -31,54 +31,46 @@ def game_teams_by_season(season)
     game_teams
   end
 
-  def total_games_per_coach(season)
-    total = Hash.new(0)
-    game_teams_by_season(season).each do |game_team|
-      total[game_team.head_coach] += 1.0
-    end
-    total
+  def winningest_coach(season)
```

Comment:
This updated version of winningest_coach is running a lot faster. We can come back and do some further refactoring by pulling out helper methods used in this method and #worst_coach